### PR TITLE
profile add profileTime

### PR DESCRIPTION
**Pull request by even986025158**
## What type of PR is this：
- [ ] enhancement

## Problem Summary(Required) ：
add Profile Time to profile

Since some executions will not be very stable due to the profile collection output after the profile is turned on, I hope to count the profile execution time.

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 4 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|0|4|00.00%|

**Review by imay — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ public class ProfileManager {
     public static final String SQL_STATEMENT = "Sql Statement";
     public static final String USER = "User";
     public static final String DEFAULT_DB = "Default Db";
+    public static final String PROFILE_TIME = "Profile Time";
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java:64]: "Profile Time" seems a little ambiguous. Would better use `Collect Profile Time` or `Compute Profile Time`

**Comment by even986025158** [comment]
> #### [FE PR Coverage check]
> 😞 **fail** : 0 / 4 (00.00%)
> 
> #### file detail
> path	covered line	new line	coverage
> 🔵	com/starrocks/qe/StmtExecutor.java	0	4	00.00%

What does it mean ？

**Comment by imay** [comment]
> > #### [FE PR Coverage check]
> > 😞 **fail** : 0 / 4 (00.00%)
> > #### file detail
> > path	covered line	new line	coverage
> > 🔵	com/starrocks/qe/StmtExecutor.java	0	4	00.00%
> 
> What does it mean ？

This is coverage checker, to tell you the coverage for your new added codes.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 4 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|0|4|00.00%|

**Comment by even986025158** [comment]
> > > #### [FE PR Coverage check]
> > > 😞 **fail** : 0 / 4 (00.00%)
> > > #### file detail
> > > path	covered line	new line	coverage
> > > 🔵	com/starrocks/qe/StmtExecutor.java	0	4	00.00%
> > 
> > 
> > What does it mean ？
> 
> This is coverage checker, to tell you the coverage for your new added codes.

and what need I do?

**Comment by kangkaisen** [comment]
> > > > #### [FE PR Coverage check]
> > > > 😞 **fail** : 0 / 4 (00.00%)
> > > > #### file detail
> > > > path	covered line	new line	coverage
> > > > 🔵	com/starrocks/qe/StmtExecutor.java	0	4	00.00%
> > > 
> > > 
> > > What does it mean ？
> > 
> > 
> > This is coverage checker, to tell you the coverage for your new added codes.
> 
> and what need I do?

@even986025158 you needn't do anything.  only rebasing the latest main code is enough.

**Review by imay — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 4 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|0|4|00.00%|

**Review by murphyatwork — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Comment by liuyehcf** [comment]
@Mergifyio backport branch-2.2

**Comment by liuyehcf** [comment]
@Mergifyio backport branch-2.3

**Pull request merged**